packages/node/src/dev-server.ts

```diff
@@ -197,8 +197,14 @@ async function compileUserCode(entrypoint: string) {
           }
 
           let response = await edgeHandler(event.request, event);
+          let responseBlob = await response.blob();
 
-          return event.respondWith(response);
+          response.headers.set('content-length', responseBlob.size.toString());
```

Comment:
This is going to break response streaming.
Next version of `edge-runtime` will remove the `content-length` header, as we're always streaming the data in local env anyway in `runtime.createServer` and in Next.js: https://github.com/vercel/edge-runtime/commit/1fa8e94de79e2fa11bd7d338ec4ff42bcd685c23

can we support response streaming in `vc dev` too? In any case, the lack of `content-header` will mean it's going to be re-calculated implicitly, right?

(there's no way to know whether the given body was a stream or not, at least without patching the `Response` constructor... which we might do in the future, but right now we don't)

Reply:
I updated this to do the same header deletes. I'm still not seeing it stream the response properly and I'm not sure what's happening here.

In the current version of this PR, if you visit http://localhost:3000/api/edge-return-fetch-binary you'll see a broken media player. If you visit http://localhost:3000/static/bunny.mp4 you'll see the video start playing.